### PR TITLE
Adding flavor map to model selector args

### PR DESCRIPTION
**Pull request by Anubha98**
Adding flavor map to model selector args

### CODE REVIEW

File: assets/training/finetune_acft_hf_nlp/src/model_selector/model_selector.py [hunk 1]
```diff
@@ -247,6 +247,26 @@ class ModelImportConstants:
 }
 
 
+FLAVOR_MAP = {
```
Comment (jpmann) [inline assets/training/finetune_acft_hf_nlp/src/model_selector/model_selector.py:250]: As discussed, add a task to flow these parameters from component script.

File: assets/training/finetune_acft_hf_nlp/src/model_selector/model_selector.py [hunk 2]
```diff
@@ -480,6 +500,9 @@ def main():
         azureml_pkg_denylist_logging_patterns=LOGS_TO_BE_FILTERED_IN_APPINSIGHTS,
     )
 
+    # Adding flavor map to args
+    args.__setattr__("flavor_map", FLAVOR_MAP)
```
Comment (jpmann) [inline assets/training/finetune_acft_hf_nlp/src/model_selector/model_selector.py:504]: nit: we can use `setattr(args, "flavor_map", FLAVOR_MAP)`

File: assets/training/finetune_acft_hf_nlp/src/model_selector/model_selector.py [hunk 3]
```diff
@@ -421,7 +441,7 @@ def model_selector(args: Namespace):
                     mlflow_data = yaml.safe_load(fp)
                 if mlflow_data and "flavors" in mlflow_data:
                     for key in mlflow_data["flavors"]:
-                        if key in ["hftransformers", "hftransformersv2"]:
```
Comment (jpmann) [inline assets/training/finetune_acft_hf_nlp/src/model_selector/model_selector.py:424]: Is this same logic applicable to OSS flavor as well? Are we going to save the same parameters in OSS MLModel file as we are doing for hftransformers MLModel file?

Comment (Anubha98) [reply assets/training/finetune_acft_hf_nlp/src/model_selector/model_selector.py:424]: yes

File: assets/training/finetune_acft_hf_nlp/src/model_selector/model_selector.py (hunk 2 above)
Comment (Anubha98) [reply assets/training/finetune_acft_hf_nlp/src/model_selector/model_selector.py]: Thanks, Fixed

**Review by jpmann — APPROVED** [review]

**Pull request merged**